assets/sass/colors/_functional.scss

```diff
@@ -8,3 +8,12 @@ $aplus-info: #1d7792; // This is a darkened version of the bootstrap 3 default.
 $aplus-warning: #a6670e; // This is a darkened version of the bootstrap 3 default. Contrast must be >=4.5 with white.
 $aplus-danger: #a50000; // There should be a contrast ratio above 7 between this and white.
 $aplus-focus: #ffdd00; // This should be clearly different fron those defned above and the brand colours.
+
+// This colours may be used to represent the status of the grading.
+// The best color to use in order to have a good contrast level should be #000.
+$aplus-primary-status-bg: #42a5f5;
+$aplus-default-status-bg: #c5c5c5;
+$aplus-success-status-bg: #8bc34a;
+$aplus-warning-status-bg: #ffb74d;
+$aplus-danger-status-bg: #ff5c72;
+$aplus-info-status-bg: #00bcd4;
```

Comment:
If we add new variables for these colours, then my initial reaction is that the variable names should be semantically correct. "Danger" is correct in the sense that it is dangerous to have failed submissions that do not pass point requirements, but it is more accurate to say "not passed" or "failed". Thus, the variable name could be `$aplus-failed-status` (value red colour).

Similarly,
* `aplus-full-points-status` (green)
* `aplus-passed-status` (yellow)
* `aplus-not-submitted-status` (gray)
* `aplus-unofficial-status` (gray)

Is it necessary to include `bg` (background) in the variable name at all? Do we have other variables for the text colour or is it always black?

What do you think of these? If we use semantic variable names now, then the same variables would maybe make more sense in the new points progress component too (#646).

Reply:
We agreed that at the moment we can keep using the Bootstrap names "success", "danger" etc. They are consistent with the CSS class names that are used everywhere in the HTML and the Bootstrap progress bar component also uses these names in the Sass variables for the progress bar colours.

Reply:
As discussed over the phone. We will keep the current names since the names somehow follow the bootstrap naming convention. Regarding the #646, if needed we can rename the variable names and use semantic names for the status of the grades.